src/poliastro/plotting/_base.py

```diff
@@ -42,7 +42,7 @@ def plane(self):
     def _set_attractor(self, attractor):
         if self._attractor is None:
             self._attractor = attractor
-        elif attractor is not self._attractor:
+        elif attractor is not self._attractor and attractor != self._attractor:
```

Comment:
Do we need to check for identity + equality? Bodies are supposed to be singletons so in my mind the identity check should be enough, but perhaps I'm missing something?

Reply:
This was something I had in my tree. I didn't want to submit it with this PR. However, I had to add it to my tree because we load some orbits from a pickle file and in that case the id() does not match but the objects are equal. The error otherwise are very strange.

Reply:
Oh, I see how that could go wrong. We can discuss this in a separate PR I think.

Reply:
I'd rather revert this change and seek a more robust solution, see https://github.com/poliastro/poliastro/issues/1395